Chapter15/E_StatusCodePagesWithRedirectRazorPages/StatusCodePagesWithRedirectRazorPages/Program.cs

```diff
@@ -2,22 +2,18 @@
 
 // Add services to the container.
 builder.Services.AddRazorPages();
-builder.Services.AddHsts(options =>
-{
-    options.MaxAge = TimeSpan.FromHours(1);
-});
 
 var app = builder.Build();
 
-
-if(app.Environment.IsProduction())
-{
-    app.UseHsts();
-}
+//Note that using this approach we're ultimately returning a success code
```

Comment:
(Chore) Add a space after the back slash makes comment looks more consistent with the existing comments 😃